dhctl/pkg/config/deckhouse_config.go

```diff
@@ -127,6 +108,18 @@ func PrepareDeckhouseInstallConfig(metaConfig *MetaConfig) (*DeckhouseInstaller,
 		return nil, fmt.Errorf("Support for 'configOverrides' was removed. Please use ModuleConfig's instead.")
 	}
 
+	if metaConfig.DeckhouseConfig.ReleaseChannel != "" {
+		return nil, fmt.Errorf("Support for 'releaseChannel' was removed. Please use 'deckhouse' ModuleConfig's settings instead.")
```

Comment:
Could we can error message more clear? Something like:

```suggestion
		return nil, fmt.Errorf("Support for 'releaseChannel' in InitConfiguration was removed. Please use 'deckhouse' ModuleConfig's settings instead.")
```